### ISSUE TITLE
Smaller Subset Data Available?

### ISSUE THREAD

**Issue by riven314** [mihaidusmanu/d2-net#46]
I am trying to run through your code for training. (Because I would like to inspect the data structure of training data, especially how correspondence points are represented and stored.)

Due to limitation of my disk space, I downloaded a subset of data (selected subfolders) from your Google Drive, but I have difficulty downloading the whole MegaDepth v1 Dataset. It is too big.

Do you have a small subset of MegaDepth data available? If so, would you mind sharing them to us for fast experimentation? Thank you!

**Comment by mihaidusmanu** [comment]
Hello. Sorry for the late reply, I somehow missed this issue. Let me know if you still need the subset of depth-maps. 

Regarding the data structure for training, all you need is :
- images with intrinsics (pinhole camera model from COLMAP) and extrinsics
- depth maps

The corresponding points are not stored anywhere in the dataset, but rather computed at each training step by the function below:
https://github.com/mihaidusmanu/d2-net/blob/2a4d88fbe84961a3a17c46adb6d16a94b87020c5/lib/loss.py#L293-L340

**Comment by riven314** [comment]
thanks! @mihaidusmanu 
it helps!
